pkg/kfapp/aws/aws.go

```diff
@@ -483,13 +421,19 @@ func (aws *Aws) Generate(resources kftypes.ResourceEnum) error {
 		return errors.WithStack(err)
 	}
 
-	// TODO: AWS doesn't enable BasicAuth yet.
-	if aws.kfDef.Spec.UseBasicAuth {
+	if pluginSpec.GetEnableBasicAuth() {
```

Comment:
em.. I am thinking if we should think it's basic auth or equivalent as coginito and oidc.
Technically, I think they are same. However, it's kind of a simple auth without extra AWS services.